apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -40,26 +58,22 @@ class GqlWithKnexLoadList {
         let skip = 0
         let newchunk = []
         let all = []
-        let haveNotifiedAboutTooManyObjs = false
+        let tooManyReturnedMemory = []
 
         let maxiterationsCount = 100 // we need some limits - 100K records is more then enough
         do {
             newchunk = await this.loadChunk(skip)
             all = all.concat(newchunk)
             skip += newchunk.length
 
-            if (!haveNotifiedAboutTooManyObjs && all && Array.isArray(all) && all.length > 1000) {
-                logger.warn({
-                    msg: 'tooManyReturned',
-                    functionName: 'GqlWithKnexLoadList.load',
-                    schemaName: this.listKey,
-                    data: {
-                        limit: 1000,
-                        GqlWithKnexLoadListArgs: { singleRelations: this.singleRelations, multipleRelations: this.multipleRelations, where: this.where, fields: this.fields },
-                    },
-                })
-                haveNotifiedAboutTooManyObjs = true
-            }
+            logTooManyReturnedIfRequired(tooManyReturnedMemory, all, {
```

Comment:
You should also introduce this change to `loadListByChunks` it is in the same file